koku/api/tags/serializers.py

```diff
@@ -17,13 +17,16 @@
 AZURE_FILTER_OP_FIELDS = ["subscription_guid"]
 GCP_FILTER_OP_FIELDS = ["account", "gcp_project"]
 OCI_FILTER_OP_FIELDS = ["payer_tenant_id"]
+day_list = ["-10", "-30", "-90"]
+month_list_string = ["-1", "-2", "-3"]
+month_list = [-1, -2, -3]
```

Comment:
If `month_list_string` is simple a str type list of `month_list` values, then I'd suggest this so the values only need to be maintained in one place.

```suggestion
month_list = [-1, -2, -3]
month_list_string = [str(m) for m in month_list]
```